couchbase/content.md

```diff
@@ -0,0 +1,258 @@
+```
+
+To configure Couchbase Server:
+
+* Access the Couchbase Server Admin Console via port 8091 on any of the hosts.
+* Follow the same steps from the *Multiple containers on single host* section, however use the use the host IP address itself rather than using `docker inspect` to discover the IP address.
+
+
+## Multiple hosts, multiple containers per host
+
+```
+┌─────────────────────────────────────────┐  ┌─────────────────────────────────────────┐
+│            Host OS (Linux)              │  │            Host OS (Linux)              │
+│ ┌─────────────────┐ ┌─────────────────┐ │  │ ┌─────────────────┐ ┌─────────────────┐ │
+│ │  Container OS   │ │  Container OS   │ │  │ │  Container OS   │ │  Container OS   │ │
+│ │    (CentOS)     │ │    (CentOS)     │ │  │ │    (CentOS)     │ │    (CentOS)     │ │
+│ │  ┌───────────┐  │ │  ┌───────────┐  │ │  │ │  ┌───────────┐  │ │  ┌───────────┐  │ │
+│ │  │ Couchbase │  │ │  │ Couchbase │  │ │  │ │  │ Couchbase │  │ │  │ Couchbase │  │ │
+│ │  │  Server   │  │ │  │  Server   │  │ │  │ │  │  Server   │  │ │  │  Server   │  │ │
+│ │  └───────────┘  │ │  └───────────┘  │ │  │ │  └───────────┘  │ │  └───────────┘  │ │
+│ └─────────────────┘ └─────────────────┘ │  │ └─────────────────┘ └─────────────────┘ │
+└─────────────────────────────────────────┘  └─────────────────────────────────────────┘
+```
+
+* Difficult to achieve with plain vanilla Docker, as there is no native way to allow each container unrestricted access to the internal IPs of containers running on other hosts.
+* There are software networking layers such as [Flannel](https://github.com/coreos/flannel "Flannel") and [Weave](https://github.com/weaveworks/weave "Weave"), but it is beyond the scope of this README to cover how those might be configured.
+* This is not a particularly useful deployment scenario for either testing or production use; you are better off checking out the various [cloud hosting scenarios](https://github.com/couchbase/docker/wiki#container-specific-cloud-hosting-platforms) available.
+
+## Cloud environments
+
+Although it is beyond the scope of this README, there is a [github wiki](https://github.com/couchbase/docker/wiki#container-specific-cloud-hosting-platforms) that contains guidance and instructions on how to run Couchbase Server Docker containers in various cloud environments. 
+
+
+# Licensing
+
+Couchbase Server comes in two editions:
+
+* [Community Edition](http://www.couchbase.com/community) -- free for unrestricted use.
+
+* [Enterprise Edition](http://www.couchbase.com/agreement/subscription) -- free for development, paid subscription required for production deployment.
+
+By default, the `latest` docker tag points to the latest Enterprise Edition, as does the `enterprise` docker tag.  If you want the Community Edition instead, you should add the `community` tag, which will point to the latest available Community Edition.
+
+```
+docker run couchbase/server:community
+```
```

Comment:
Delete this "Licensing" section from `content.md` and move L254-258 to the `license.md` as that is used to generate the "License" section in `README.md` (which is why there is two setcions in `README.md`).